doc/io/dms/standalone.md

```diff
@@ -1,26 +1,36 @@
 # AWS DMS Standalone
 
 ## About
-Relay an AWS DMS data stream from Amazon Kinesis into a [CrateDB] table using
-a one-stop command `ctk load table kinesis+dms://...`.
 
-You can use it in order to facilitate convenient data transfers to be used
-within data pipelines or ad hoc operations. It can be used as a CLI interface,
-and as a library.
+Relay an [AWS DMS] data stream from Amazon Kinesis into a [CrateDB] table using
+a one-stop command `ctk load table kinesis+dms:///...`.
```

Comment:
"Into a table" is clearly wrong, because a DMS event stream may converge to multiple outbound tables. Also thanks for mentioning this, @hammerhead.

Reply:
The wording has been fixed, included with 7b56477996.